Client/sdk/game/CModelInfo.h

```diff
@@ -154,6 +154,9 @@ class CModelInfo
     virtual CVector      GetVehicleDummyPosition(eVehicleDummies eDummy) = 0;
     virtual void         SetVehicleDummyPosition(eVehicleDummies eDummy, const CVector& vecPosition) = 0;
     virtual void         ResetVehicleDummies(bool bRemoveFromDummiesMap) = 0;
+    virtual float        GetVehicleWheelSize(eResizableVehicleWheelGroup eWheelGroup) = 0;
+    virtual void         SetVehicleWheelSize(eResizableVehicleWheelGroup eWheelGroup, float fWheelSize) = 0;
+    virtual void         ResetVehicleWheelSizes(std::pair<float, float>* defaultSizes = NULL) = 0;
```

Comment:
Should this be `nullptr` instead?

Reply:
Yeah. I just let the old me decide to use that NULL.